examples/6-advanced-flows.html

```diff
@@ -0,0 +1,213 @@
+            "You tap or click the best responses."
+        ],
+        "reply": [
+            {
+                "question": "Play On!",
+                "answer": "chapter-one"
+            }
+        ]
+    },
+    "chapter-one": {
+        "says": [
+            "Excellent -- Thank you.",
+            "[Questions | Info | Requests for feedback]",
+        ],
+        "reply": [
+            {
+                "question": "I do!",
+                "answer": "chapter-two" // here both replies send people onto the same next chapter 
+            },
+            {
+                "question": "I do not.",  // both replies send people onto the same next chapter 
+                "answer": "chapter-two"
+            },
+            {
+                "question": "Repeat the question, please? [Context | Instructions]" ,
+                "answer": "intro-context"
+            }
+        ],
+    },
+    "chapter-two": {
+        "says": [
+            "Very good. Have you ...",
+            "[Question to filter out respondents | Create a branch in the logic]",
+        ],
+        "reply": [
+            {
+                "question": "I have.",
+                "answer": "chapter-three"
+            },
+            {
+                "question": "I have not. [Escape Valve hit | End]",
+                "answer": "end" // An example of using the Escape Valve for ineligible, Not Applicable participants 
+            },
+        ]
+    },
+    "chapter-three": {
+        "says": [
+            "Go ahead to the next chapter.",
+            "<img src=http://meanbusiness.com/wp-content/uploads/2018/03/Coral-Participate-Phone_and_Laptop.jpg />",
+        ],
+        "reply": [
+            {
+                "question": "OK, I will.", // A Gimme: An easy answer with fewer choices to keep the egagement going. Useful during longer dialogues
+                "answer": "chapter-four"
+            }
+        ]
+    },
+    "chapter-four": {
+        "says": [
+            "Thank you very much.",
+            "<img src=http://meanbusiness.com/wp-content/uploads/2018/02/mel_b-Thank-You.gif />",
+            "More? Or Quit while we're behind?",
+        ],
+        "reply": [
+            {
+                "question": "Quit [Escape Valve hit]",
+                "answer": "end"
+            },
+            {
+                "question": "More",
+                "answer": "chapter-five"
+            },
+        ],
+    },
+    "chapter-five": {
+        "says": [
+            "[Answering 'Yes' here will open a new window with a page from another site]", // call a function to an external resource / application outside of chat-bubble's purview
+        ],
+        "reply": [
+            {
+                "question": "Not interested, thanks. [Escape Valve hit]",
+                "answer": "end"
+            },
+            {
+                "question": "Yes",
+                "answer": "externalResourceFunction"
```

Comment:
I think this function should also give user an option to continue chat. Otherwise the window will pop up and there will be no way to continue from here on.

Reply:
Will add that.

Reply:
I've addressed all but this one and can use guidance re getting start over / ice to work within the script.